src/utils/set.ts

```diff
@@ -7,6 +7,7 @@ module Plottable {
      * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
      */
     export class Set<T> {
+      public size = 0;
```

Comment:
We could try to add some safety by doing this every time we want to set `size`:
```Typescript
Object.defineProperty(this, "size", { configurable: true, writable: false, value: newSize });
```
... although people could still mutate `size` by doing the same thing. The above code keeps people from doing something like `set.size++` or `set.size = 9001`, though.